lib/ui/painting.dart

```diff
@@ -4259,6 +4268,124 @@ class FragmentProgram extends NativeFieldWrapperClass1 {
   external Handle _shader(_FragmentShader shader, Float32List floatUniforms, List<ImageShader> samplerUniforms);
 }
 
+/// A [Shader] generated from a [FragmentProgram].
+///
+/// Instances of this class can be obtained from the `fragmentShader()` method
+/// of a [FragmentProgram] instance. The [floatUniforms] list is initialized to
```

Comment:
Maybe worth pointing out that these are zero filled already

Reply:
Done.